tests/integration-tests.yaml

```
@@ -0,0 +1,47 @@
+kind: Pipeline
+apiVersion: tekton.dev/v1beta1
+metadata:
+  name: example-pipeline
```

Please give a more explanatory name.
```suggestion
  name: trustee-operator-integration-tests-pipeline
```